diff --git a/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle b/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
index bcf9b00..9384440 100644
--- a/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
+++ b/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
@@ -82,7 +82,7 @@ ext.useExpoPublishing = {
     publishing {
       publications {
         release(MavenPublication) {
-          from components.release
+          afterEvaluate { from components.findByName("release") }
         }
       }
       repositories {
